src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -242,6 +246,168 @@ namespace VariableSpeedCoils {
 		VarSpeedCoil.deallocate();
 	}
 
+	// Default Constructor
```

Comment:
[It is here](https://github.com/NREL/EnergyPlus/blob/fed09fcb8f94a93050a35c40b18bfb3f87b2e383/src/EnergyPlus/VariableSpeedCoils.cc#L164) but really I moved this because of `AutoSize` since I didn't want to `#include <DataHVACGlobals.hh>` or `#include <DataSizing.hh>` (I forget which) in the header file. I am trying to minimize the number of `#include` and `using` in header files.